invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/card.overrides

```diff
@@ -65,18 +65,27 @@
 }
 
 .community-frontpage-cards .ui.card {
-
   .image {
-    min-height: 12rem;
-    max-width: 12rem;
-    margin: auto;
+    height: 16rem;
+    display: flex;
+    align-items: center;
     background: transparent;
-    position: relative;
+    overflow: hidden;
```

Comment:
`overflow: hidden` will potentially cut the logo, no?

Reply:
if the logo has unusal dimensions (very long or wide ) it will probably be cut off yes, but honestly if you would try to show a logo like without cutting it off,  it will usually become so small (to make it fit either horizontally or vertically) that it would bother more imo.

Reply:
In the last screenshot you added, the placeholder is also cut, i.e. a logo with the same dimensions as the placeholder will also get cut on that screen size?

Reply:
On the biggest screen dimension the cards become wider but not longer so yes it has been cut a bit :sweat_smile:

Reply:
It's the same for some smaller screens and tall rectangles, no? 

Imo it's more important that the logos are not cut, than them filling the entire container.